docker/deploy/Dockerfile

```diff
@@ -0,0 +1,73 @@
+ARG BUILD_ENV
+
+FROM $BUILD_ENV as builder
+
+ENV MKL_LIB=/opt/intel/compilers_and_libraries/linux/mkl/lib/intel64
```

Comment:
I recommend to
`source /opt/intel/compilers_and_libraries/linux/mkl/bin/mklvars.sh intel64`
to define MKL's environment and use
`MKL_LIB=$MKLROOT/lib/intel64`

Reply:
Ah yes, I think I have tried it previously, but then I had not set the shell to bash for it to work yet